docs: add docs for cache size suggestion (#1125)
juicedata/juicefs-csi-driver: docs/en/guide/resource-optimization.md

## Set reasonable cache size for Mount Pod {#set-reasonable-cache-size-for-mount-pod}

> [Node-pressure eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction) is usually set in Kubernetes cluster, `nodefs.available` is available disk space on either the node's root filesystem. JuiceFS default cache size is 100GiB, default cache directory's minimum free space ratio `free-space-ratio` is 0.1, and the default cache size is very likely to trigger node eviction. It's recommended to set a reasonable cache size according to the actual disk space of the node.

Reviewer: ```suggestion
[Node-pressure eviction](https://kubernetes.io/docs/concepts/scheduling-eviction/node-pressure-eviction) is usually set in a Kubernetes cluster. `nodefs.available` is the available disk space of the node's root file system. The default cache size of JuiceFS is 100GiB. `free-space-ratio`, the minimum free space ratio of the default cache directory, is 0.1. The default cache size is likely to trigger node eviction. It is recommended to set a reasonable cache size according to the actual disk space of the node.
```
